File: app/src/main/java/org/ole/planet/myplanet/callback/OnCourseItemSelected.java
package org.ole.planet.myplanet.callback;

import org.ole.planet.myplanet.model.RealmMyCourse;
import org.ole.planet.myplanet.model.RealmTag;

import java.util.List;

public interface OnCourseItemSelected {
    void onSelectedListChange(List<RealmMyCourse> list);
    void onTagClicked(RealmTag tag);
}

File: app/src/main/java/org/ole/planet/myplanet/callback/OnCourseItemSelected.kt
package org.ole.planet.myplanet.callback

import org.ole.planet.myplanet.model.RealmMyCourse
import org.ole.planet.myplanet.model.RealmTag

interface OnCourseItemSelected {
    @JvmSuppressWildcards
    fun onSelectedListChange(list: List<RealmMyCourse>)
    fun onTagClicked(tag: RealmTag)
}
